cli/README.md

```diff
@@ -21,34 +22,54 @@ Optional:
 
 ## Workspace Setup
 
-Go is very particular about the layout of a source tree, and the naming of packages.  It is therefore important to 
-get the code from github.com/apache/brooklyn-client/cli and not your own fork. If you want to contribute to the 
-project, the procedure to follow is still to get the code from github.com/apache/brooklyn-client/cli, and then to add your
-own fork as a remote. 
+Go is very particular about the layout of a source tree and the source repository, 
+as it relies on this in the naming of packages.  
+Developers familiar with [`$GOPATH`](http://golang.org/cmd/go/#hdr-GOPATH_environment_variable) -- or on OS's that don't support soft links --
```

Comment:
I'd add an explanation about what the reason for the difference here is, something like 

> If you just want to develop the `br` tool itself you may simply work in your usual GOPATH, using `go get github.com/apache/brooklyn-client/cli/br` and adding your own fork as a remote. `br` is built just like any other Go project.
> 
> If you want to clone the Apache Brooklyn projects, including `brooklyn-client`, and work on both the CLI and the rest of the Brooklyn code,  then you may prefer to avoid having one copy of the CLI code in your Apache Brooklyn workspace and a separate copy in your GOPATH for `br` development.  This can be done quite simply by following the instructions below, to link the `brooklyn-client` code in your GOPATH to your regular Apache Brooklyn workspace.
> 
> First ensure that your `GOPATH` is set, for example... (etc, as below)